components/authentication/authUtils.js

```diff
@@ -116,15 +126,17 @@ const userValidation = async (password = '', userName = '') => {
   return errorCollection;
 };
 
-const alphanumRegex = /^[0-9a-zA-Z]{6,}$/i;
+const logoutUser = async () => {
+  const res = await axios.get(`${process.env.NEXT_PUBLIC_API_URL}${'/logout'}`);
+  return res.data;
+};
 
-const userCheck = userName => {
-  if (userName.length < 6) {
-    return false;
-  } else {
-    if (alphanumRegex.test(userName)) return true;
-    else return false;
-  }
+const registerUser = async (newUserSchema) => {
+  const res = await axios.post(
+    `${process.env.NEXT_PUBLIC_API_URL}${'/register'}`,
+    newUserSchema
+  );
+  return res;
```

Comment:
Are we handling errors elsewhere? If not add a try catch block at the level where errors for `login` and `logout` are being handled.

Reply:
yes they are being handled when its being called